xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -1108,6 +1108,34 @@ void CDVDPlayer::CheckBetterStream(CCurrentStream& current, CDemuxStream* stream
     OpenStream(current, stream->iId, stream->source);
 }
 
+int CDVDPlayer::PreloadFileInfo(const CFileItem& file, const CPlayerOptions &options)
+{
+  m_bAbortRequest = false;
+  SetPlaySpeed(DVD_PLAYSPEED_NORMAL);
+
+  m_State.Clear();
```

Comment:
this is no go. m_State and friends are not supposed to be modified from outside.